object/service.proto

```diff
@@ -156,6 +156,15 @@ service ObjectService {
   //   provided session token has expired.
   rpc Search(SearchRequest) returns (stream SearchResponse);
 
+  // Search for objects in a container. Similar to Search, but:
+  // * sorted
```

Comment:
the current proposal does not provide way to skip sorting and cursor calculation. Perhaps the costs will be insignificant, but still the idea came. Anyway, this can be done later if needed

Reply:
That's exactly the feature. Sorting _is_ important (you can't properly merge results otherwise) and cursor doesn't cost a lot.

Reply:
im not a supporter of transferring potentially ignored data, but given their small size, let it be